config/logstash.yml

```diff
@@ -70,17 +70,17 @@
 # available to plugins that implement an ECS Compatibility mode for use with
 # the Elastic Common Schema.
 # Possible values are:
-# - disabled
-# - v1 (default)
-# - v2
```

Comment:
missed this in docs first time around, the setting has accepted `v8` for a while.